modules/SofaExporter/src/SofaExporter/WriteState.inl

```diff
@@ -302,7 +302,7 @@ void WriteState::handleEvent(sofa::core::objectmodel::Event* event)
             // store the actual time instant
             lastTime = d_time.getValue()[nextIteration];
             // if the time simulation is >= that the actual time instant
-            if ( (time > lastTime) || (fabs(time - lastTime)< std::numeric_limits<double>::epsilon()) )
+            if ( (time > lastTime) || (fabs(time - lastTime)< std::numeric_limits<float>::epsilon()) )
```

Comment:
Why ?

Reply:
because it was not working...
I will test it again

Reply:
Here is what I get if I put back epsilon in double:
WriteState: time: 4.95
WriteState: lastTime: 4.95
WriteState: fabs(time - lastTime): 9.76996e-15
WriteState: std::numeric_limits<double>::epsilon(): 2.22045e-16

and it doesn't enter the if. But it should.
Any suggestion?